<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>지균이와 감자들</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../사진 및 그림/icon.png</normaloff>../사진 및 그림/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #FFFFFF;</string>
  </property>
  <widget class="QLabel" name="program_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vesta</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../사진 및 그림/톱니바퀴.png</normaloff>../사진 및 그림/톱니바퀴.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="loopAni">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Loop Animaition</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hidden_mic_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;  /* 테두리 제거 */
    background-color: transparent;  /* 배경을 투명하게 설정 */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>280</width>
     <height>360</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>278</width>
      <height>358</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="command_edit">
       <property name="styleSheet">
        <string notr="true">border: 2px solid gray; border-radius: 10px; padding: 5px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="answer_edit">
       <property name="styleSheet">
        <string notr="true">border: 2px solid gray; border-radius: 10px; padding: 5px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>scrollArea</zorder>
  <zorder>program_name</zorder>
  <zorder>toolButton</zorder>
  <zorder>loopAni</zorder>
  <zorder>hidden_mic_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
